SourceX/controls/game_controls.cpp

```diff
@@ -142,6 +147,38 @@ bool GetGameAction(const SDL_Event &event, ControllerButtonEvent ctrl_event, Gam
 		default:
 			break;
 		}
+		if (dpad_hotkeys) {
+			switch (ctrl_event.button) {
+			case ControllerButton_BUTTON_DPAD_UP:
+				if (IsControllerButtonPressed(ControllerButton_BUTTON_BACK))
+					*action = GameActionSendKey{ DVL_VK_F6, ctrl_event.up };

```

Comment:
I was wondering why you duplicated the Select + A/B/X/Y mappings then I had a look at the Vita's button layout and it made sense.

Reply:
Yeah. That was requested by users.